apps/accounts/forms/settings.py

```diff
@@ -34,11 +40,11 @@ def clean_email(self):
             raise forms.ValidationError('This email is already registered')
         return email
 
-    def save(self, domain=None, commit=True):
+    def save(self, domain=None, commit=True, confirmed=False):
```

Comment:
Change lines 28-32 signature/body to:

```
    def __init__(self, user=None, require_verification=True, *args, **kwargs):
        super(AddEmailForm, self).__init__(*args, **kwargs)
        self.user = user
        self.require_verification = require_verification
        set_input_attrs(self)
        set_input_placeholder_labels(self)
```

Get rid of the additional parameter here (`confirmed=False`)